File: src/com/zypex/piflow/DrivetrainConfig.java
package com.zypex.piflow;

public class DrivetrainConfig{

    public double maxVelocity;
    public double maxAcceleration;
    public double maxJerk;

    public double turnSpeed = 1;

    public PIDFConstants velocityPID;

    public DrivetrainConfig(double maxVelocity, double maxAcceleration, double maxJerk) {
        this.maxVelocity = maxVelocity;
        this.maxAcceleration = maxAcceleration;
        this.maxJerk = maxJerk;
    }
}

File: src/com/zypex/piflow/DrivetrainConfig.kt
package com.zypex.piflow

import com.zypex.piflow.PIDFConstants

class DrivetrainConfig(var maxVelocity: Double, var maxAcceleration: Double, var maxJerk: Double) {
    var turnSpeed = 1.0
    var velocityPID: PIDFConstants? = null
}